### ISSUE TITLE
Nucleus update 2.12.2 -> 2.12.5

### ISSUE THREAD

**Issue by tomikpik** [aws-greengrass/aws-greengrass-nucleus#1621]
Hi there, I'd like to perform a remote upgrade of the Nucleus from 2.12.2 to 2.12.5. 

The update ends up with the following exception:
```
com.aws.greengrass.deployment.exceptions.DeploymentException: Nucleus update workflow failed to restart Nucleus. See loader logs for more details
	at com.aws.greengrass.deployment.KernelUpdateDeploymentTask.getDeploymentStatusDetails(KernelUpdateDeploymentTask.java:132)
	at com.aws.greengrass.deployment.KernelUpdateDeploymentTask.call(KernelUpdateDeploymentTask.java:76)
	at com.aws.greengrass.deployment.KernelUpdateDeploymentTask.call(KernelUpdateDeploymentTask.java:36)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

Unfortunately, I am not able to locate the mentioned "loader logs" to diagnose it further and there is nothing unusual in the greengrass.log Turning the DEBUG logging does not make a difference.

Could you please help me to diagnose it further?

Thank you

**Comment by MikeDombo** [comment]
Please look for logs from SystemD using `journalctl -u greengrass`, this is the loader log. 

Also look at the log file named aws.greengrass.Nucleus

**Comment by junfuchen99** [comment]
Hello, Nucleus v2.12.5 has a bug where it could cause a crash at startup on certain ARMv8 processors. It might be the issue you were running into and we have fixed the issue in Nucleus v2.12.6.
